tensorboard/webapp/widgets/data_table/types.ts

```diff
@@ -129,3 +129,5 @@ export interface AddColumnEvent {
   nextTo?: ColumnHeader | undefined;
   side?: Side | undefined;
 }
+
+export type ColumnGroup = 'RUN' | 'EXPERIMENT_ALIAS' | 'HPARAM' | 'OTHER';
```

Comment:
nit: I think this should be an enum as it is a bit more consistent with the rest of our code (though I personally do like string literals)

```suggestion
export enum ColumnGroup {
    RUN,
    EXPERIMENT_ALIAS,
    HPARAM,
    OTHER
}
```

Reply:
Done! I'll prefer enums going forward.